### PR TITLE
Latest go lang version

### PR DESCRIPTION
**Pull request by Rishi-web**
w.r.t issue - https://github.ibm.com/kms/kmk/issues/1426

Upgrading go lang to latest version in travis.yml file

### CODE REVIEW

**Comment by Rishi-web** [comment]
Screenshot 2022-09-29 at 11 31 02 AM

File: .travis.yml [hunk 1]
```diff
@@ -2,8 +2,8 @@ language: go
 dist: xenial
 
 go: 
- - 1.14.x
- - 1.15.x
+ - 1.18.x
```
Comment (dinesh-venkatraman-g) [inline .travis.yml:9]: We should continue to support 1.14 & 1.15 as well

**Review by dinesh-venkatraman-g — CHANGES_REQUESTED** [review]

**Comment by Rishi-web** [comment]
Screenshot 2022-09-29 at 5 05 08 PM

**Comment by Rishi-web** [comment]
Screenshot 2022-09-29 at 5 05 59 PM

**Review by dinesh-venkatraman-g — CHANGES_REQUESTED** [review]
Update the travis job to support support last 3 version of go lang.

**Review by dinesh-venkatraman-g — APPROVED** [review]

**Pull request merged**